src/apexpy/helpers.py

```diff
@@ -166,15 +166,16 @@ def subsol(datetime):
         year = np.asanyarray([datetime.year])
         doy = np.asanyarray([datetime.timetuple().tm_yday])
         ut = np.asanyarray([datetime.hour * 3600 + datetime.minute * 60
-                            + datetime.second])
+                            + datetime.second + datetime.microsecond / 1e6])
```

Comment:
```suggestion
                            + datetime.second + datetime.microsecond / 1.0e6])
```

To make clear that this should be float division and not integer division.